src/estimators/exact_set.py

```diff
@@ -84,7 +100,10 @@ def __call__(self, sketch_list):
     """Return len(sketch) - 1."""
```

Comment:
Can we update the comment here to explain that we're returning cardinality short by 1 for all frequency?

Reply:
Done.